cpp/src/parquet/arrow/reader_internal.cc

```diff
@@ -353,7 +353,8 @@ Status TransferBool(RecordReader* reader, MemoryPool* pool, Datum* out) {
 }
 
 Status TransferInt96(RecordReader* reader, MemoryPool* pool,
-                     const std::shared_ptr<DataType>& type, Datum* out) {
+                     const std::shared_ptr<DataType>& type, Datum* out,
+                     const ::arrow::TimeUnit::type& int96_arrow_time_unit) {
```

Comment:
You do not need to pass `TimeUnit::type` as a reference, since it's a cheap trivial type. Just pass it by value.

Reply:
Addressed.